<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>426</width>
    <height>299</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>381</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pb10">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>90</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb100">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>150</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>100</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb500">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>180</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>500</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb50">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>120</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>50</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbCoffee">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>90</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Coffee(100)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbTea">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>120</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Tea(150)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbCoke">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>150</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Coke(200)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbReset">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>180</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <zorder>lcdNumber</zorder>
   <zorder>pb10</zorder>
   <zorder>pb100</zorder>
   <zorder>pb500</zorder>
   <zorder>pb50</zorder>
   <zorder>pbCoke</zorder>
   <zorder>pbReset</zorder>
   <zorder>pbTea</zorder>
   <zorder>pbCoffee</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>426</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
